bc/subscription/utils/courtlistener.py

```diff
@@ -64,6 +74,13 @@ def get_docket_id_from_query(query: str) -> int:
     validator = URLValidator()
     validator(cleaned_str)
 
+    # check if the query string is a PDF link
+    is_pdf_link = re.search(PDF_URL_PATTERN, cleaned_str)
+    if is_pdf_link:
+        r = requests.get(is_pdf_link.group("url_for_redirect"))
```

Comment:
Don't forget a timeout on all requests. https://github.com/psf/requests/issues/3070.

```suggestion
        url = is_pdf_link.group("url_for_redirect")
        r = requests.get(url, timeout=10)
```

Reply:
You're right! I will add the timeout.